app/src/main/java/org/mozilla/fenix/tabstray/ext/TabSessionState.kt

```diff
@@ -5,6 +5,23 @@
 package org.mozilla.fenix.tabstray.ext
 
 import mozilla.components.browser.state.state.TabSessionState
+import mozilla.components.browser.toolbar.MAX_URI_LENGTH
+import mozilla.components.lib.publicsuffixlist.PublicSuffixList
+import org.mozilla.fenix.ext.toShortUrl
+
+/**
+ * Shortens URLs to be more user friendly, by applying [String.toShortUrl]
+ * and making sure it's equal or below the [MAX_URI_LENGTH].
+ */
+fun TabSessionState.toShortUrl(publicSuffixList: PublicSuffixList): String {
```

Comment:
Can you change this to a @Composable function and use `inComposePreview` so that we can avoid passing in `PublicSuffixList` by calling `LocalContext.components.publicSuffixList` or something